trusted-committer-article.md

```diff
@@ -0,0 +1,296 @@
+# InnerSource Learning Path: Trusted Committer
+
+## Introduction
+
+- tbd
+
+## Introducing The Trusted Committer Role
+
+The Trusted Committer (TC) role is one of the key roles in an InnerSource
+community. Think of TCs as a person in a community that you trust with
+important technical decisions that will also mentor contributors to get their
+contribution over the finish line. It is both a demanding and a rewarding
+role to fulfill. It goes far beyond being an opinionated gatekeeper and it is
+instrumental for the success of any InnerSource community. 
+
+Generally speaking, the TC role is defined by its responsibilities, rather than
+by its privileges. On a very high level, TCs represents the interests of both
+their InnerSource community and the products the community is building. They
+are concerned with the health of the community and the product. So as a TC,
+you'll have both tech oriented and community oriented responsibilities and
+we'll explore both of these dimensions in the following sections. 
+
+Before we go into the details of what a TC actually does, let's spend some time
+contrasting the TC role to other roles in IS and why we think the name is both
+apt and important.
+
+- Contributor (very briefly)
+    - makes contributions
+    - sometimes part of the community
+    - sometimes sent by another part of the organization to develop a feature
+      for it. This is why contributors are sometimes referred to as being part
+      of the _Guest Team_. 
+    - responsible for "fitting in"/conforming to the host team and it's 
+      expectations and processes
+    - similar to an Apache Developer or committer
+
+- Trusted Committer
+    - In the analogy above, a member of the _Host Team_.
+    - In this analogy, they are both responsible for building the house and
+      setting the house rules to make sure their guests are comfortable.
+    - In comparison to the Contributor, a TC has earned the responsibility to
+      push code closer to production and is generally allowed to perform tasks
+      that have a higher level of risk associated with them.
+- Product Owner
+    - Responsible for defining and prioritizing requirements and stories for
+      the community to implement.
+    - Also interacts directly and frequently with the trusted committers. E. g.
+      a TC will want to collaborate with the PO to make sure donated features
+      actually belong into the product. 
+    - Generally speaking, the TCs role is not to accept all contributions but
+      only those which meet the criteria in terms of scope and quality.
+    - Distinction between the guest and host PO - have to negotiate
+      requirements, if necessary.
+    - Depending on the type of InnerSource community, there might not be a PO
```

Comment:
Could mention here that it's common for a lead TC to take on the role of PO in these bottom-up situations.